src/recursionAssignment/Recursion.java

```diff
@@ -1,5 +1,66 @@
 package recursionAssignment;
 
-public class Recursion{
+import java.io.*;
 
+public class Recursion {
+  /**
+  Prints a a random number 
```

Comment:
include @param specifications for your javadoc comments to indicate parameters for your methods.